envoy/http/filter.h

```diff
@@ -105,6 +105,19 @@ enum class FilterHeadersStatus {
   StopAllIterationAndWatermark,
 };
 
+/**
+ * Return codes for encode on informative headers filter invocations.
+ */
+enum class Filter1xxHeadersStatus {
+  // Continue filter chain iteration.
+  Continue,
+  // Do not iterate for informative headers on any of the remaining filters in the chain.
+  //
+  // Returning FilterDataStatus::Continue from encodeData() or calling
```

Comment:
FilterDataStatus::Continue? copy paste error?

Reply:
This unfortunately is the existing behavior. Messaged out of band , happy to handle this in a follow up to keep this as a clean refactor.